src/test/java/com/karankumar/bookproject/ui/components/utils/ComponentUtilTest.java

```diff
@@ -0,0 +1,42 @@
+package com.karankumar.bookproject.ui.components.utils;
```

Comment:
Could you please attach our [copyright notice](https://github.com/Project-Books/book-project/blob/master/docs/COPYRIGHT) to the start of this file?

Reply:
Done